src/main/java/com/ontotext/kafka/service/SinkRecordsProcessor.java

```diff
@@ -132,16 +144,28 @@ public Object call() throws RetriableException {
 		long start = System.currentTimeMillis();
 		try (RepositoryConnection connection = repository.getConnection()) {
 			connection.begin();
+			int recordsInCurrentBatch = recordsBatch.size();
+			if (LOG.isTraceEnabled()) {
+				LOG.trace(
+						"Transaction started in GraphDB Repository connection {} , Batch size: {} , Records in current batch: {}",
+						this.repositoryUrl, batchSize, recordsInCurrentBatch);
+			}
 			while (!recordsBatch.isEmpty()) {
 				SinkRecord record = recordsBatch.remove();
 				if (!failedRecords.contains(record)) {
 					handleRecord(record, connection);
 				}
 			}
 			connection.commit();
+			if (LOG.isTraceEnabled()) {
```

Comment:
Why LOG.isTraceEnabled(), it makes sense to use it when you have additional operations that are related to logging the trace info. But for simple logging makes no sense.